core/start-blueos-core

```diff
@@ -31,7 +31,7 @@ SERVICES=(
     'commander',"$SERVICES_PATH/commander/main.py"
     'nmea_injector',"$SERVICES_PATH/nmea_injector/nmea_injector/main.py"
     'helper',"$SERVICES_PATH/helper/main.py"
-    'video',"mavlink-camera-manager --default-settings BlueROVUDP --mavlink tcpout:127.0.0.1:5777 --gst-feature-rank omxh264enc=0,v4l2h264enc=250,x264enc=260 --log-path /var/logs/blueos/mavlink-camera-manager --verbose"
+    'video',"mavlink-camera-manager --default-settings BlueROVUDP --mavlink tcpout:127.0.0.1:5777 --gst-feature-rank omxh264enc=0,v4l2h264enc=250,x264enc=260 --log-path /var/logs/blueos/mavlink-camera-manager --vehicle-ddns blueos.local --verbose"
```

Comment:
Will that break for people that have no mdns server running on their machines ? Unable to find `blueos.local` on the network.

Reply:
> Will that break for people that have no mdns server running on their machines ? Unable to find `blueos.local` on the network.

Yes, it will :o

Reply:
I was not aware we were supporting setups without MDNS, I can remove that commit :cry:

Reply:
removed.